client/blocks/reader-site-notification-settings/index.jsx

```diff
@@ -115,14 +123,23 @@ class ReaderSiteNotificationSettings extends Component {
 					className="reader-site-notification-settings__button"
 					onClick={ this.togglePopoverVisibility }
 					ref={ this.spanRef }
+					aria-label={ translate( 'Notification settings' ) }
 				>
-					<Gridicon icon="cog" size={ 24 } ref={ this.iconRef } />
-					<span
-						className="reader-site-notification-settings__button-label"
-						title={ translate( 'Notification settings' ) }
-					>
-						{ translate( 'Settings' ) }
-					</span>
+					<SVGIcon
+						classes="reader-following-feed"
+						name="settings"
+						size={ this.props.iconSize }
+						icon={ Settings }
+						ref={ this.iconRef }
+					/>
+					{ this.props.showLabel && (
+						<span
+							className="reader-site-notification-settings__button-label"
+							title={ translate( 'Notification settings' ) }
+						>
+							{ translate( 'Settings' ) }
+						</span>
+					) }
 				</button>
 
 				<ReaderPopover
```

Comment:
If you want to review the follow setting button changes outside of the feed/site stream, you can go directly to `/following/manage` and look for "Settings" to the right of each followed site.